python/GafferUI/Slider.py

```diff
@@ -206,6 +214,64 @@ def getPositionIncrement( self ) :
 
 		return self.__positionIncrement
 
+	## Sets whether mouse interactions with the slider should snap to
+	# specific position increments. This does not apply to programmatic
+	# changes via setPosition(s) or keyboard increments.
+	def setSnaps( self, snaps ) :
+
+		self.__snaps = snaps
+
+	def getSnaps( self ) :
+
+		return self.__snaps
+
+	def setSnapIncrement( self, snapIncrement ) :
```

Comment:
Alternatively, we could have `snapIncrement is None` be the way of signalling that snapping is off?

Reply:
Switched to an increment of `None` meaning no snapping in rework.